docs/src/solvers/dae_solve.md

```diff
@@ -26,6 +26,18 @@ extra options for the solvers, see the ODE solver page.
 - `ROS3P` - 3rd order A-stable and stiffly stable Rosenbrock method. Keeps high
   accuracy on discretizations of nonlinear parabolic PDEs.
 - `Rodas3` - 3rd order A-stable and stiffly stable Rosenbrock method.
+- `RosShamp4`- An A-stable 4th order Rosenbrock method.
+- `Veldd4` - A 4th order D-stable Rosenbrock method.
+- `Velds4` - A 4th order A-stable Rosenbrock method.
```

Comment:
Let's make sure these others added to the mass matrix tests @YingboMa

Reply:
Luckily, ROS4 and ROSW are included in the macros. We only need to check `Rosenbrock23/32` class and `Rodas` class.

Reply:
Those already have tests. The other ones, like RosShamp4, I don't think are in the mass matrix tests

Reply:
https://github.com/JuliaDiffEq/OrdinaryDiffEq.jl/pull/862

Reply:
Awesome. It looks like Rosenbrock23 should be added to this list as well